.buildkite/hooks/pre-command

```diff
@@ -25,8 +25,15 @@ if [ -d "/tmp/artifacts" ]; then
     find /tmp/artifacts -print || true
 fi
 
-echo "Creating var/ artifact directory..."
-docker run --rm -v /tmp/artifacts:/artifact-mount alpine:latest /bin/sh -c 'chown -R 2000 /artifact-mount/' || true
+echo "Creating artifact directory..."
```

Comment:
move the echo into the if too?

Reply:
the echo applies for both if clauses (linux vs. windows)